core/standalone/start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+docker run --rm -d \
+  -h openwhisk --name openwhisk \
+  -p 3233:3233 -p 3232:3232 \
+  -v //var/run/docker.sock:/var/run/docker.sock \
+ ${1:-openwhisk}/standalone
+sleep 5
```

Comment:
May be bump it to say 10. On My Mac it did not came up in 5 sec and saw a blank page

Reply:
Yes it takes a bit of time I know. Also some time even if you open the playground it fails running actions. Eveything is fine after a while. It would be useful to have a readyness check somewhere but I dunno how to do it.

Reply:
you can curl the playground apis and wait for a response.